examples/logger.py

```diff
@@ -85,11 +117,21 @@ def __init__(self, name: str, parent: Optional[QObject] = None):
         self.loggerFrame.clearButton.clicked.connect(self.checkToClear)
         self.confirmFrame = ConfirmClearingFrame()
         self.confirmFrame.confirmed.connect(self.clearLog)
+        # define handler
```

Comment:
You don't have to write obvious comments (the code immediately tells you that a handler is defined).